core/quantity/src/test/Parsing.test.ts

```diff
@@ -215,7 +210,7 @@ describe("Parsing tests:", () => {
       { value: "-2FT 6IN", quantity: { magnitude: -2.5, unitName: "Units.FT" } },
       { value: "1 1/2 FT", quantity: { magnitude: 1.5, unitName: "Units.FT" } },
       { value: "2FT 6IN", quantity: { magnitude: 2.5, unitName: "Units.FT" } },
-      { value: `2'-6"`, quantity: { magnitude: 2.5, unitName: "Units.FT" } },
+      { value: `2' 6"`, quantity: { magnitude: 2.5, unitName: "Units.FT" } },
```

Comment:
whats the change here for?

Reply:
Previously, the minus sign was implicitely threated like a [formatTraits fractionDash](https://www.itwinjs.org/bis/ec/ec-format/#:~:text=the%20rounding%20factor.-,fractionDash,-Use%20a%20dash). 
Now, with the minus sign, `fractionDash` must be explicitely added in the format traits. Else it's implicitely an operator.

Reply:
It should implicitly be an operator, if you have set allowMathematicalOperations right?

Reply:
Well, is the user trying to do a subtraction or is he using a fraction dash ?   
My take is that, since it's ambiguous, we should return an error if neither is chosen. 

I think that supporting fractionDash by default was a bug. Because it acted differently that other formatTraits. 
Else, other traits should also be active by default, for consistency.